packages/auth/src/env.ts

```diff
@@ -0,0 +1,5 @@
+import { createEnvironment, variable } from "@dotkomonline/environment"
+
+export const env = createEnvironment({
+  RPC_HOST: variable,
+})
```

Comment:
This should not depend on it, but the app that's calling getAuthOptions should provide ity instead